comp/api/api/apiimpl/internal/config/endpoint.go

```diff
@@ -25,7 +25,7 @@ const prefixPathSuffix string = "."
 type authorizedSet map[string]struct{}
 
 var authorizedConfigPathsCore = buildAuthorizedSet(
-	"api_key", "site", "dd_url", "logs_config.dd_url", "ha",
+	"api_key", "site", "dd_url", "logs_config.dd_url",
```

Comment:
Confidence check: no `mrf` needs here?

Reply:
This was old code based on when we were trying the original ADP approach for HAMR, and needed to call the Agent internal API to get HAMR configuration settings.

Since this is entirely within the Agent now, we have no need to expose any of that here.